app/views/govuk_publishing_components/components/_layout_header.html.erb

```diff
@@ -28,18 +27,22 @@
         <div class="govuk-grid-column-full govuk-grid-column-one-third-from-desktop gem-c-layout-header__search">
           <%= render "govuk_publishing_components/components/layout_header/search" %>
         </div>
-        <div class="govuk-header__content gem-c-header__content govuk-grid-column-full">
-          <%= render "govuk_publishing_components/components/layout_header/navigation_items", navigation_items: navigation_items, navigation_aria_label: navigation_aria_label  %>
-        </div>
+        <% if navigation_items.any? %>
```

Comment:
Not necessary within the scope of this PR, but do we need any temporary selector for slimmer to be able to render the login for accounts?

Reply:
Not that I can see in the [accounts_shower.rb](https://github.com/alphagov/slimmer/blob/971af9aff9d26b8e549fc739f05e8afdeca0a9c3/lib/slimmer/processors/accounts_shower.rb) - both the logged in and logged out links need to be [added using the `navigation_items` parameter](https://github.com/alphagov/static/blob/master/app/views/root/_base.html.erb#L35-L59), and then Slimmer chooses which to show and hide.